apps/daimo-mobile/src/logic/linkStatus.ts

```diff
@@ -18,6 +18,16 @@ export function useFetchLinkStatus(
   return rpcHook.getLinkStatus.useQuery({ url: url! }, { enabled: !!url });
 }
 
+export function fetchLinkStatus(
+  link: DaimoLink | undefined,
+  daimoChain: DaimoChain
+) {
+  const sanitizedLink = link && stripSeedFromNoteLink(link);
+  const url = sanitizedLink && formatDaimoLink(sanitizedLink);
+  const rpcFunc = getRpcFunc(daimoChain);
+  return rpcFunc.getLinkStatus.query({ url: url! });
```

Comment:
recommend not using `!` > bypasses ts checks

why is the link passed in optional / what's the desired behavior for `fetchLinkStatus(undefined)`?

can say

```
if (url == null) return null;
return rpcFunc...
```